bundles/SeoBundle/src/Model/Redirect.php

```diff
@@ -154,7 +154,11 @@ public function getTarget(): ?string
     public function getTargetPath(): string
     {
         $redirectTarget = $this->target;
-        $targetDocumentPath = Document::getById($this->target)?->getFullPath();
+        $targetDocumentPath = null;
+
+        if (is_numeric($this->target)) {
+            $targetDocumentPath = Document::getById((int)$this->target)?->getFullPath();
+        }
```

Comment:
```suggestion
        if (is_numeric($redirectTarget)) {
            $targetDocumentPath = Document::getById((int)$redirectTarget)?->getFullPath();
        }
```

Or maybe even better: get rid of the `$redirectTarget` variable (GitHub somehow doesn't allow me to create a suggestion for this).

Reply:
Hey @jdreesen I also thought about that, but I would say it is not scope of this task. We can do another PR for this. Maybe you want to create a new PR for this yourself?